lavamoat/browserify/main/policy.json

```diff
@@ -2238,6 +2238,66 @@
         "document.defaultView": true
       }
     },
+    "@rive-app/react-canvas>@rive-app/canvas": {
+      "globals": {
+        "AudioContext": false,
```

Comment:
Note for other reviewers: these policy.json updates are set to `false` because policy-override is merged into the resulting policy at the time it's generated. 
They would have been set to true otherwise (so the change to override file is necessary)